src/aics-image-viewer/components/App/index.tsx

```diff
@@ -952,202 +540,277 @@ export default class App extends React.Component<AppProps, AppState> {
-          collapsed={this.state.controlPanelClosed}
-          width={500}
-        >
-          <ControlPanel
-            showControls={showControls}
-            getMetadata={() => this.getMetadata()}
-            // image state
-            imageName={this.state.image?.name}
-            hasImage={!!this.state.image}
-            pixelSize={this.state.image ? this.state.image.pixel_size : [1, 1, 1]}
-            channelDataChannels={this.state.image?.channels}
-            channelGroupedByType={this.state.channelGroupedByType}
-            // user selections
-            pathTraceOn={viewerSettings.renderMode === RenderMode.pathTrace}
-            channelSettings={this.state.channelSettings}
+  useImageEffect((_image) => view3d.setShowAxis(viewerSettings.showAxes), [viewerSettings.showAxes]);
+
+  useImageEffect(
+    (_image) => view3d.setBackgroundColor(colorArrayToFloats(viewerSettings.backgroundColor)),
+    [viewerSettings.backgroundColor]
+  );
+
+  useImageEffect(
+    (image) => view3d.setBoundingBoxColor(image, colorArrayToFloats(viewerSettings.boundingBoxColor)),
+    [viewerSettings.boundingBoxColor]
+  );
+
+  useImageEffect(
+    (image) => view3d.setShowBoundingBox(image, viewerSettings.showBoundingBox),
+    [viewerSettings.showBoundingBox]
+  );
+
+  useImageEffect(
+    (image) => {
+      const { renderMode } = viewerSettings;
+      view3d.setMaxProjectMode(image, renderMode === RenderMode.maxProject);
+      view3d.setVolumeRenderMode(renderMode === RenderMode.pathTrace ? RENDERMODE_PATHTRACE : RENDERMODE_RAYMARCH);
+      view3d.updateActiveChannels(image);
+    },
+    [viewerSettings.renderMode]
+  );
+
+  useImageEffect(
+    (image) => {
+      view3d.updateMaskAlpha(image, alphaSliderToImageValue(viewerSettings.maskAlpha));
+      view3d.updateActiveChannels(image);
+    },
+    [viewerSettings.maskAlpha]
+  );
+
+  useImageEffect(
+    (_image) => {
+      const isPathTracing = viewerSettings.renderMode === RenderMode.pathTrace;
+      const brightness = brightnessSliderToImageValue(viewerSettings.brightness, isPathTracing);
+      view3d.updateExposure(brightness);
+    },
+    [viewerSettings.brightness]
+  );
+
+  useImageEffect(
+    (image) => {
+      const isPathTracing = viewerSettings.renderMode === RenderMode.pathTrace;
+      const density = densitySliderToImageValue(viewerSettings.density, isPathTracing);
+      view3d.updateDensity(image, density);
+    },
+    [viewerSettings.density]
+  );
+
+  useImageEffect(
+    (image) => {
+      const imageValues = gammaSliderToImageValues(viewerSettings.levels);
+      view3d.setGamma(image, imageValues.min, imageValues.scale, imageValues.max);
+    },
+    [viewerSettings.levels]
+  );
+
+  useImageEffect(
+    (image) => view3d.setInterpolationEnabled(image, viewerSettings.interpolationEnabled),
+    [viewerSettings.interpolationEnabled]
+  );
+
+  useImageEffect(
+    (image) => view3d.setVolumeTranslation(image, props.transform?.translation || [0, 0, 0]),
```

Comment:
again, it makes me nervous when this `image` has the same name as the state `image`